mambaSharedFramework/Pantos-Generic HLS Playlist Parsing/Pantos-Generic Tag Validators/EXT_X_SESSION_KEYValidator.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  EXT_X_SESSION_KEYValidator.swift
+//  mamba
+//
+//  Created by Robert Galluccio on 8/31/24.
+//  Copyright © 2024 Comcast Corporation.
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License. All rights reserved.
+//
+
+import Foundation
+
+// All attributes defined for the EXT-X-KEY tag (Section 4.4.4.4) are also defined for the
+// EXT-X-SESSION-KEY, except that the value of the METHOD attribute MUST NOT be NONE.
+class EXT_X_SESSION_KEYValidator: EXT_X_KEYValidator {
```

Comment:
since you're using a super class method maybe this one can just be `final class`?

Reply:
Makes sense... I'm even on the fence of whether I want to keep this as a class extension. Maybe I change this to a struct and take a similar approach as I did with `EXT_X_SESSION_DATATagValidator` where I instantiate a local generic validator, and instantiate a local `EXT_X_KEYValidator` (to favor composition) 🤔

Reply:
Updated the approach here: c871de1e2999716474282d8d8706622ca2581dfb

What do you think?